src/Raven.Client/ServerWide/DatabaseTopology.cs

```diff
@@ -310,7 +345,7 @@ public bool RelevantFor(string nodeTag)
             return destinations;
         }
 
-        public bool EntireDatabasePendingDeletion(Dictionary<string, DeletionInProgressStatus> deletionInProgress)
+        public virtual bool EntireDatabasePendingDeletion(Dictionary<string, DeletionInProgressStatus> deletionInProgress, DatabaseTopology[] _)
```

Comment:
let pass the `RawDatabaseRecord` here instead

Reply:
`RawDatabaseRecord` is server code and `DatabaseTopology` is client. And I'd rather not pass `DatabaseRecord` because cluster observer will have to materialize it just for the shard topologies.

Reply:
let's remove this from here and move this to the updater? (need to mark this as a breaking change)